src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.querySelector('tr');
+
+rows.forEach((row) => {
+  const cells = row.querySelector('td, th');
+
+  if (cells.length > 2) {
```

Comment:
Since `cells` is expected to be a NodeList (if `querySelectorAll` is used), you should check its length using `cells.length`.